cirq-google/cirq_google/experimental/noise_models/noise_models.py

```diff
@@ -65,6 +67,22 @@ def __init__(
         self.bitflip_probs = bitflip_probs
         self.decay_probs = decay_probs
 
+    def _value_equality_values_(self):
+        return (
+            tuple(sorted((q, p) for q, p in (self.depol_probs or {}).items())),
+            tuple(sorted((q, p) for q, p in (self.bitflip_probs or {}).items())),
+            tuple(sorted((q, p) for q, p in (self.decay_probs or {}).items())),
```

Comment:
Those dictionaries have simple float values.  Would it work with just using them as are?

Reply:
Resolved in 2c93e87ee47d9eab237b08d2ee79844bce105d77.  These attributes can be None which has a bit different effect than `{}`, so they are stored as None if such.